### PR TITLE
Blog post: Supabase Beta December 2022

### PR DESCRIPTION
**Pull request by RamiroND**
## What kind of change does this PR introduce?

New Blog post: Supabase Beta December 2022

### CODE REVIEW

File: apps/www/_blog/2023-01-05-supabase-beta-december-2022.mdx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+thumb: 2023-01-05-december-beta-update/monthly-update-december2022.jpg
+tags:
+  - release-notes
+date: '2023-01-05'
+toc_depth: 3
+---
+
+Happy New Year! Hope you are off to a great start. We closed 2022 on a high note with [Launch Week 6](https://supabase.com/launch-week) and a cascade of announcements. Here is everything we shipped so you can try them all 💻
+
+## Day 1: New Supabase Docs
```
Comment (awalias) [inline apps/www/_blog/2023-01-05-supabase-beta-december-2022.mdx:15]: Should we take "Supabase" out of the titles? Seems redundant

**Review by awalias — DISMISSED** [review]

File: apps/www/_blog/2023-01-05-supabase-beta-december-2022.mdx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+- Edgy Edge Functions: a look at a new Edge Function per week. [Youtube Playlist](https://www.youtube.com/playlist?list=PL5S4mPUpp4OulD3olUW8Eq1IYKpUbk5Ob)
+- Supabase Angular authentication with RxJS Observables. [Tutorial](https://dev.to/kylerummens/supabase-angular-authentication-with-rxjs-observables-3d0f)
+- Create Calendar Events in React With Google Calendar API and Supabase. [Video Tutorial](https://www.youtube.com/watch?v=tgcCl52EN84
+- Going Serverless with Supabase. [Part 1](Going "Serverless" With Supabase)
+- Supabase a Complete Backend Solution. [Video](https://www.youtube.com/watch?v=Ve0qE55fw0Y)
+- Supabase OAuth with SvelteKit (Discord, Google, GitHub). [Video](https://www.youtube.com/watch?v=KfezTtt2GsA)
+
+## Meme Zone
+
+As always, one of our favorite memes from last mont. [Follow us on Twitter](https://twitter.com/supabase) for more.
```
Comment (dannykng) [inline apps/www/_blog/2023-01-05-supabase-beta-december-2022.mdx:116]:
```suggestion
As always, one of our favorite memes from last month. [Follow us on Twitter](https://twitter.com/supabase) for more.
```

File: apps/www/_blog/2023-01-05-supabase-beta-december-2022.mdx [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+## Postgres Ecosystem
+
+We are all in on Postgres. That is why contributing to the ecosystem is always a priority. Launch Week 6 was no exception, with several announcements:
+
+![Postgres Ecosystem](/images/blog/2023-01-05-december-beta-update/postgres-ecosystem.jpg)
+
+- [pg_graphql v1.0](https://supabase.com/blog/pg-graphql-v1)
+- [PostgREST 11 pre-release](https://supabase.com/blog/postgrest-11-prerelease)
+- [What's new in Postgres 15?](https://supabase.com/blog/new-in-postgres-15)
+- [pg_crdt - an experimental CRDT extension for Postgres](https://supabase.com/blog/postgres-crdt)
```
Comment (dannykng) [inline apps/www/_blog/2023-01-05-supabase-beta-december-2022.mdx:77]: This list doesn't seem to be rendering correctly: https://zone-www-dot-com-git-beta-december-supabase.vercel.app/blog/supabase-beta-december-2022#postgres-ecosystem

Comment (dannykng) [inline apps/www/_blog/2023-01-05-supabase-beta-december-2022.mdx:77]:
```suggestion
- [pg_graphql v1.0](https://supabase.com/blog/pg-graphql-v1)
- [PostgREST 11 pre-release](https://supabase.com/blog/postgrest-11-prerelease)
- [What's new in Postgres 15\?](https://supabase.com/blog/new-in-postgres-15)
- [pg_crdt - an experimental CRDT extension for Postgres](https://supabase.com/blog/postgres-crdt)
```

Comment (dannykng) [reply apps/www/_blog/2023-01-05-supabase-beta-december-2022.mdx:77]: Seems like the `?` in the link is throwing it off.

File: apps/www/_blog/2023-01-05-supabase-beta-december-2022.mdx (hunk 1 above)
Comment (RamiroND) [reply apps/www/_blog/2023-01-05-supabase-beta-december-2022.mdx:15]: Makes sense from this one and Storage.

**Review by dannykng — APPROVED** [review]

**Pull request merged**